crates/biome_js_analyze/src/lint/nursery/use_import_extensions.rs

```diff
@@ -19,7 +19,11 @@ declare_rule! {
     /// Tooling also benefits from explicit file extensions, because they do not need to guess which
     /// file to resolve.
     ///
-    /// Rule checks static imports and dynamic imports calls such as `import()` and `require()`.
+    /// The rule checks static imports and dynamic imports calls such as `import()` and `require()`.
+    ///
+    /// To ensure that the TypeScript LSP adds the file extension when it automatically imports a variable,
```

Comment:
```suggestion
    /// To ensure that the LSP adds the file extension when it automatically imports a variable,
```

Maybe remove the `TypeScript` because this applies to `JavaScript` too. Or change `LSP` to `editor`?